release.nix

```diff
@@ -1,7 +1,7 @@
 let
   fixedNixpkgs = (import ./lib.nix).fetchNixPkgs;
 in
-  { supportedSystems ? [ "x86_64-linux" "x86_64-darwin" ]
+  { supportedSystems ? [ "x86_64-linux" "x86_64-darwin" "win64" ]
```

Comment:
Maybe we want to be more explicit here in the future? To specify that "win64" is only a `supportedTarget`?
Lumping it together with the other `supportedSystems`, might suggest (and confuse) that it's built on "win64"?